client-v2/src/services/__tests__/faciaApi.spec.ts

```diff
@@ -75,22 +76,43 @@ describe('faciaApi', () => {
       }
     });
   });
-  describe('getArticles', () => {
-    it('should issue a CAPI request for an array of article ids', () => {
+  describe('getArticlesBatched', () => {
+    it('should issue a CAPI request for an array of article ids', async () => {
       fetchMock.once(
-        getCapiUriForArticleIds(['article1', 'article2']),
+        getCapiUriForContentIds(['article1', 'article2']),
         '{"response":{ "results": [] }}'
       );
-      getArticles(['article1', 'article2']);
+      await getArticlesBatched(['article1', 'article2']);
     });
-    it('should chunk requests for large numbers of articles into separate requests', () => {
-      const articleIds = [...Array.from(Array(175).keys()).map(_ => _.toString())];
+    it('should chunk requests for large numbers of articles into separate requests', async () => {
+      const articleIds = [
+        ...Array.from(Array(175).keys()).map(_ => _.toString())
+      ];
       const chunkedArticleIds = chunk(articleIds, 50);
-      chunkedArticleIds.map(_ => fetchMock.once(
-        getCapiUriForArticleIds(_),
-        '{"response":{ "results": [] }}'
-      ));
-      getArticles(articleIds);
+      chunkedArticleIds.map(_ =>
+        fetchMock.once(
+          getCapiUriForContentIds(_),
+          '{"response":{ "results": [] }}'
+        )
+      );
+      await getArticlesBatched(articleIds);
+    });
+  });
+  describe('getArticles', () => {
```

Comment:
rename `getArtcles` function to singular and maybe instead of articles get content since it is no longer getting articles?

Reply:
done and done 👍